WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -46,10 +47,17 @@ typedef NS_ENUM(NSInteger, PostSettingsRow) {
     PostSettingsRowExcerpt
 };
 
+typedef NS_ENUM(NSInteger, PostMetaRow) {
+    PostMetaRowAuthor = 0,
+    PostMetaRowPublishDate,
+    PostMetaRowPublishStatus,
+    PostMetaRowVisibility,
+    PostMetaRowPassword
+};
```

Comment:
Nice to see these indices in an enum ❤️

Reply:
Now that we've gone with the new approach of dynamically generating the list of rows, I wonder if this new enum is really needed, wdyt? I think we could just as easily use the existing enum which has a case for each row.

Reply:
Totally agree! Deleted code is best code. Thanks for spotting this.